dependencies/default/pom.xml

```diff
@@ -44,6 +44,7 @@
     <commons-io.version>2.11.0</commons-io.version>
     <commons-lang.version>2.6</commons-lang.version>
     <commons-lang3.version>3.12.0</commons-lang3.version>
+    <commons-text.version>1.9</commons-text.version>
```

Comment:
we sort dependencies by alphabeta, it should below `commons-logging`

Reply:
fixed